connectors/config.py

```diff
@@ -0,0 +1,42 @@
+#
+# Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+# or more contributor license agreements. Licensed under the Elastic License 2.0;
+# you may not use this file except in compliance with the Elastic License 2.0.
+#
+
+import os
+
+from envyaml import EnvYAML
+
+from connectors.logger import logger
+
+config = None
+
+
+def load_config(config_file):
+    global config
```

Comment:
The way it's implemented now, it seems like anyone can use `config` variable if the module is imported. It's similar to the singleton approach in potential harm, but in this case anyone who imports the module can actually modify the content of `config`.

I think it should not be globally accessible.

Reply:
Indeed. I will make it just a function to load config, and make it only called from `cli`, so all the services will take config as a param.